### PR TITLE
manuscript edits from dan

### PR DESCRIPTION
**Pull request by dschride**
Here are my suggested edits (and hopefully some new typos) to the manuscript. I didn't touch the responses as they look fine to me.

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -505,7 +507,8 @@ \subsubsection*{ Single-population demographic models.}
 model it is also influenced by migration between the target population and populations not analyzed in inference.
 Since ancestral population sizes correspond to the inverse of the coalescence rates, by computing the
 inverse coalescence rates from the simulated model, we obtain the best estimates
-one may hope to infer by analyzing a single population. We thus analytically compute inverse coalescence
+one may hope to infer by analyzing a single population with a method that seeks to match coalescence time distributions.
+. We thus analytically compute inverse coalescence
```
Comment (grahamgower) [inline popSimManu.tex:511]: Looks like a stray full stop at the start of the line here.

Comment (dschride) [reply popSimManu.tex:511]: Yup. fixed.

File: popSimManu.tex [hunk 2]
```diff
@@ -369,6 +370,7 @@ \subsection*{The species Catalog}
 %For \emph{E. coli} we have a simple, large constant population size model implemented.
 In addition to organism-specific models, \stdpopsim also includes a generic piecewise constant size model and
 isolation with migration (IM) model which can be used with any genome and genetic map.
+%DRS: do we want to include these generic models in Table 1?
```
Comment (petrelharp) [inline popSimManu.tex:373]: nah, but either way is ok

Comment (petrelharp) [reply popSimManu.tex:373]: there'd be no citation and no timing info for them, so I think definately not

File: popSimManu.tex [hunk 3]
```diff
@@ -517,18 +520,10 @@ \subsubsection*{ Single-population demographic models.}
...
-with varying population size can be fit to data generated from a more complex model including migration.
-In this case, the coalescence time distributions
-will exactly match, and the coalescence rate is the inverse of the effective population size.
-However, in this specific model,
-the inverse coalescence rates are very close to the simulated population sizes
-(Figure \ref{fig:census_vs_IRC}).
+and black lines indicate the ``true" values depicted by the inverse coalescence rates (although in this specific model
+the inverse coalescence rates are very close to the simulated population sizes;
+Figure \ref{fig:census_vs_IRC}).
+%DRS: trying to cut down on the redundancy between the coal rate stuff here and the previous paragraph
```
Comment (petrelharp) [inline popSimManu.tex:526]: :+1:

**Review by petrelharp — APPROVED** [review]
Looks good!

**Pull request merged**